README.md

```diff
@@ -1,5 +1,59 @@
+# Bowling Challenge in JavaScript
 
-Bowling Challenge
+The project is a translation of last weeks weekend challenge which was written in Ruby to JavaScript.<br>
+The program acts as a electronic score card for a game of bowling which allows users to input the pins they<br>
+knock down in a frame frame over 2 rolls and displays the score on a per frame and culmulative basis.
+
+## Project Planning
+
+[Project Planning Draft Notes Page 1](https://github.com/Battery0/bowling-challenge/blob/main/public/planning1.jpg)
+
+[Project Planning Draft Notes Page 2](https://github.com/Battery0/bowling-challenge/blob/main/public/planning2.jpg)
+
+## To Do
+
+1. Account for spares bonus points
+2. Account for strike bonus points
+3. Include stubs & doubles to isolate tests
+4. Add test for console output on 'game over'
+
+## Getting started
```

Comment:
nice clear Readme instructions, totally forgot to add those in mine. Nice clear classes, each with a single responsibility, and the tests are clear in what they do.  I think if given some more time you could easily write scoring for spares and strikes, as you are already storing the rolls for each frame. As you mentioned doubling/stubbing I can see that you only need it for the score class really.